app/controllers/api/v1/appointments_controller.rb

```diff
@@ -0,0 +1,45 @@
+class Api::V1::AppointmentsController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    # @appointments = current_user.appointments
+    render json: current_user.appointments.includes([:technician]).order(id: :desc),
+           each_serializer: AppointmentSerializer
+  end
+  # def index
+  #   render json: current_user.bookings.includes([:technician]).order(id: :desc), status: :ok
+  # end
```

Comment:
- [ ] Kindly, remove the old commented code present in this file, that way, the repo's cleanliness will be improved. 💪